python/lsst/ts/atwhitelight/config_schema.py

```diff
@@ -0,0 +1,190 @@
+          of command_interval), though the code does try to send
+          the read watchdog command first.
+        type: number
+        exclusiveMinimum: 0
+      command_interval:
+        description: >-
+          Minimum interval (seconds) between sending each command to the
+          ThermoTek chiller. The ThermoTek manual claims this must be >= 1
+          but experiments suggest it can be much shorter.
+        type: number
+        exclusiveMinimum: 0
+      telemetry_interval:
+        description: >-
+          Interval (seconds) after one set of telemetry commands
+          finishes before queuing the next set. These commands are run
+          at low priority, so the actual interval before the next command
+          runs may be longer.
+        type: number
+        exclusiveMinimum: 0
+      watchdog_interval:
+        description: >-
+          Minimum interval (seconds) between watchdog commands.
+          Watchdog data is more important than telemetry data;
+          it includes whether the chiller is running and whether
+          there are any alarms or warnings.
+        type: number
+        exclusiveMinimum: 0
+    requiredProperties:
+      - host
+      - port
+      - initial_temperature
+      - low_ambient_temperature_warning
+      - high_ambient_temperature_warning
+      - low_supply_temperature_warning
+      - high_supply_temperature_warning
+      - low_process_flow_warning
+      - low_ambient_temperature_alarm
+      - high_ambient_temperature_alarm
+      - low_supply_temperature_alarm
+      - high_supply_temperature_alarm
+      - low_process_flow_alarm
+      - connect_timeout
+      - command_interval
+      - telemetry_interval
+      - watchdog_interval
+    additionalProperties: false
+  lamp:
+    description: Configuration for the lamp controller and the LabJack that controls it
+    type: object
+    properties:
+      device_type:
+        description: LabJack device type, e.g. T4
+        type: string
+        enum: [T4, T7]
+      connection_type:
+        description: Type of LabJack connection, e.g. TCP
+        type: string
+      identifier:
+        description: LabJack identifier; an IP host address if connection_type=TCP
+        type: string
+      connect_timeout:
+        description: Maximum time to connect and read state (seconds)
+        type: number
+        exclusiveMinimum: 0
+      cooldown_period:
+        description: >-
+          How long after turning off the lamp before you can turn the lamp
+          back on or turn off the chiller (seconds).
+          Recommended value: 900 seconds, to preserve lamp life.
+        type: number
+        exclusiveMinimum: 0
+      warmup_period:
+        description: >-
+          How long after turning on the lamp before you can adjust the power
+          or turn the lamp off without forcing it (seconds).
+          Recommended value: 900 seconds, to preserve lamp life.
```

Comment:
Discussed offline. I added an ignition timer since we're not at all convinced it is safe to turn the lamp down immediately. However, we'll try short values for that timer. This sounds like something to look up in the lamp controller manual.

Reply:
You were right. The lamp controller uses full power for a brief duration for ignition, but then backs off to the user-specified value. So I got rid of this configuration parameter.